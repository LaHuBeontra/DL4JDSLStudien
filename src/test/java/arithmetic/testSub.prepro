function main() {
    import vec3 p1, vec3 p2;

    vec3 sub = p1 - p2;

    export sub;
}
