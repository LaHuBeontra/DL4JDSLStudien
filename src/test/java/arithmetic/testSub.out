PreProDataSet{
dataSet=[-1.00,  0.00,  0.00,  -44,661.00,  0.00,  0.00,  -1.00,  0.00,  0.00,  -1.00,  0.00,  0.00]
variableNames=[sub]
variableShapes=[4, 3]}